shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiHelper.java

```diff
@@ -89,6 +97,12 @@ public static <T extends Resource> T resourceInBundle(Bundle bundle, Class<T> re
         return resourcesInBundle(bundle, resourceType).findFirst().orElse(null);
     }
 
+    // MSH-10
+    public static String getMessageControlId(Bundle bundle) {
+        var msh10 = bundle.getIdentifier().getValue();
```

Comment:
I'd update this to just return bundle.getIdentifier().getValue();  There's no need to store it in a variable first